### PR TITLE
fix: enable Basemaps preview for Elevation data TDE-1151 BM-932

### PR DESCRIPTION
**Pull request by amfage**
#### Motivation

Enable the Basemaps configuration for `dem_lerc`, as Basemaps now supports previewing Elevation data.

#### Modification

Remove exclusion for `dem_lerc` preset.

#### Checklist

- [ ] Tests updated N/A
- [x] Docs updated
- [x] Issue linked in Title

### CODE REVIEW

**Review by l0b0 — COMMENTED** [review]
The code looks good, but it would be good to understand why we excluded the DEM LERC files before, and what's changed to make us include them again.

File: workflows/raster/standardising.yaml [hunk 1]
```diff
@@ -377,7 +377,7 @@ spec:
             template: get-location
 
           - name: create-overview
-            when: "'{{workflow.parameters.target_epsg}}' =~ '2193|3857' && '{{workflow.parameters.compression}}' != 'dem_lerc'"
```
Comment (blacha) [inline workflows/raster/standardising.yaml:380]: Overviews do not work with LERC files :sob: , this step still needs to be skipped.

Comment (amfage) [reply workflows/raster/standardising.yaml:380]: Oh, so it isn't in production?

Comment (amfage) [reply workflows/raster/standardising.yaml:380]: Oh I see, the overviews. Got it.

**Comment by amfage** [comment]
> The code looks good, but it would be good to understand why we excluded the DEM LERC files before, and what's changed to make us include them again.

I have updated the PR description.

**Review by paulfouquet — APPROVED** [review]

**Pull request merged**